### ISSUE TITLE
Problem defining new resources

### ISSUE THREAD

**Issue by csvarer** [tine20/tine20#6970]
With the newest resease of the community edition, v 2018.08.6 there is a problem adding a new resource for the calendar. Using an admin account and adding a container works perfectly well (as a "Calendar_Model_resource"). It is also visible as a resource, but right clicking on the new resource doesn't show or find it as a "Resource" (nor when trying to define a new event and include the resource). Neither, as administrator when right clicking the resource in Calendar, it is possible to edit the Resource (as it with the resources that has been transferred from previous versions of tine20).

Somehow there must be an error in the newly defined resources. When trying to edit the old resources it works perfectly well in the new version 2018.08.6 of tine20, but the olddefined "resource" are not shown as "Calendar_Model_Resource" but as "Calendar_Model_Event, when using "admin->Container->edit" The new defined container has the type "Calendar_Model_Resource" but is not visible as a resource when defining new events.

Is there any way to correct for this?

**Comment by Gloirin** [comment]
Dear Csvarer,

resources are not normal containers.
please try to create new resource over Admin->Application->Calendar. Another option are over CoreData->Calendar or Addressbook->Resources.

Best regards Christian Feitl

**Comment by csvarer** [comment]
Dear Christian,

Thank you so much for your promptly response. Doing it using Admin -> Application -> Calendar works perfectly well, so it has fully solved our problem. Perhaps the other way of doing it should be [disabled? I think it was the way we defined the old room resources when we set up our system like 5 years ago.

I have just added an image to this comment showing the difference between the two ways of defining sesources (conference room is the old one, nursing room is the new one)
![diffbetweenoldandnewresource](https://user-images.githubusercontent.com/45598251/49594859-0aa7e580-f977-11e8-95ab-835dc8166716.JPG)
 - it doesn't matter as it is solved now, but just for your information

Once again - thank you very much for your help
kind regards
Claus
